ddl/column.go

```diff
@@ -1426,16 +1455,24 @@ func checkAndApplyNewAutoRandomBits(job *model.Job, t *meta.Meta, tblInfo *model
 	if err != nil {
 		return err
 	}
+	convertedFromAutoInc := mysql.HasAutoIncrementFlag(oldCol.Flag)
+	if convertedFromAutoInc {
```

Comment:
```suggestion
	if mysql.HasAutoIncrementFlag(oldCol.Flag) {
```

Reply:
I think this way improves the readability.